jelares/portfolio/src/main/webapp/javascript/blogLogic.js

```diff
@@ -0,0 +1,83 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+    Fetches the URL for uploading to Blobstore and adds it to the blog form
+*/
+function fetchBlobstoreUrl() {
+  fetch('/blobstore-upload-url')
+      .then((response) => {
+        return response.text();
+      })
+      .then((imageUploadUrl) => {
+        const messageForm = document.getElementById('blog-post');
+        messageForm.action = imageUploadUrl;
+      });
+}
+
+/*
+    Posts the actual blog content
```

Comment:
"Posts" may be confused with making a POST request. Do you mean:

```
Adds blog content to the "blog-posts" element.
```

Reply:
I made the phrasing more clear